### ISSUE TITLE
[BUG] [Formatter] `--format-js`/`--format-css` still corrupt `{# #}` template comments inside `<script>`/`<style>`

### ISSUE THREAD

**Issue by Lenormju** [djlint/djLint#2163]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
  * `v1.39.6` release today
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
  * follow-up of my previous issue
- [x] I've read the [docs](https://djlint.com)
  * follow-up of my previous issue

## System Info

- OS: Ubuntu 24.04
- Python version: CPython 3.12.3
- djLint version: 1.39.5
- template language: Jinja

## How To Reproduce

```html
<script>
{# TODO: fix later #}
var x = 1;
</script>
```

`djlint --reformat --profile jinja --format-js repro.html`

Actual result:
```
<script>
    {
         # TODO: fix later #
    }
    var x = 1;
</script>
```

Expected result: `{# TODO: fix later #}` preserved unchanged.

## Notes

* 1.39.6 (c69c8fd8118ecca6671f8c75a92c2460d0693f78) fixed the corruption of `{% %}` and `{{ }}` tags inside `<script>`/`<style>` by adding `mask_template_tags`/`restore_template_tags` around the `jsbeautifier`/`cssbeautifier` calls. However, `{# ... #}` template comments are not masked, so they are still passed to the beautifier raw and destroyed.
* my previous issue #2159 did not mention the third Jinja cases, which did not get fixed
